update-embedded

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
+var path = require('path');
+
+var input = '',
+    stats = { docs: 0, updates: 0 },
+    docPath,
+    doc;
+
+if (process.argv.length !== 3) {
+  console.error('Please provide the file system path to a doc.');
+  process.exit(1);
+} else {
+  docPath = process.argv[2];
+  doc = require(path.resolve(docPath));
+}
+
+var isObject = function (obj) {
```

Comment:
I think it's the only script that is currently using it, but if not I should probably use a library or pull it out into one.  Let me know if you have a suggestion.